src/runmode-napatech.c

```diff
@@ -69,128 +79,108 @@ void RunModeNapatechRegister(void)
 #endif
 }
 
+
 #ifdef HAVE_NAPATECH
-int NapatechRegisterDeviceStreams()
+
+
+static int NapatechRegisterDeviceStreams(void) 
 {
-    NtInfoStream_t info_stream;
-    NtInfo_t info;
-    char error_buf[100];
-    int status;
-    int i;
-    char live_dev_buf[9];
-    int use_all_streams;
-    ConfNode *ntstreams;
-    ConfNode *stream_id;
 
-    if (ConfGetBool("napatech.use-all-streams", &use_all_streams) == 0)
-    {
+    // Display the configuration mode
+    int use_all_streams;
+    if (ConfGetBool("napatech.use-all-streams", &use_all_streams) == 0) {
```

Comment:
this means that if the "napatech.use-all-streams" option is not in the yaml, suricata will exit. Is that intentional?